src/Rules/Function/IncludeFunctionRule.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\Function;
+
+use TwigCsFixer\Rules\AbstractFixableRule;
+use TwigCsFixer\Token\Token;
+use Webmozart\Assert\Assert;
+
+/**
+ * Ensures that include function is used instead of function tag.
+ */
+final class IncludeFunctionRule extends AbstractFixableRule
+{
+    protected function process(int $tokenPosition, array $tokens): void
+    {
+        $token = $tokens[$tokenPosition];
+        if (!$this->isTokenMatching($token, Token::BLOCK_NAME_TYPE, 'include')) {
+            return;
+        }
+
+        $openingTag = $this->findPrevious(Token::BLOCK_START_TYPE, $tokens, $tokenPosition);
+        Assert::notFalse($openingTag, 'Opening tag cannot not be null.');
+
+        $closingTag = $this->findNext(Token::BLOCK_END_TYPE, $tokens, $tokenPosition);
+        Assert::notFalse($closingTag, 'Closing tag cannot not be null.');
+
+        $fixer = $this->addFixableError(
+            'Include function must be used instead of include tag.',
+            $token
+        );
+
+        if (null === $fixer) {
+            return;
+        }
+
+        $fixer->beginChangeSet();
+        $fixer->replaceToken($openingTag, '{{');
+        $fixer->replaceToken($tokenPosition, 'include(');
+        if ($this->isTokenMatching($tokens[$tokenPosition + 1], Token::WHITESPACE_TOKENS)) {
+            $fixer->replaceToken($tokenPosition + 1, '');
+        }
+        $ignoreMissing = false;
+        $withoutContext = false;
+        foreach (range($tokenPosition, $closingTag - 1) as $position) {
+            $token = $tokens[$position];
+            if (Token::BLOCK_NAME_TYPE === $token->getType() || Token::NAME_TYPE === $token->getType()) {
+                switch ($token->getValue()) {
+                    case 'with':
+                        $fixer->replaceToken($position, ',');
+                        break;
+                    case 'only':
+                        $withoutContext = true;
+                        $fixer->replaceToken($position, '');
+                        break;
+                    case 'ignore':
+                        $ignoreMissing = true;
+                        $fixer->replaceToken($position, '');
+                        break;
+                    case 'missing':
+                        $fixer->replaceToken($position, '');
+                        break;
+                    default:
+                        break;
+                }
+            }
+        }
+        $endInclude = ') }}';
+        if ($withoutContext) {
+            $endInclude = ', with_context = false'.$endInclude;
+        }
+        if ($ignoreMissing) {
+            $endInclude = ', ignore_missing = true'.$endInclude;
+        }
```

Comment:
I'm not sure about named parameters. I didn't see them often used in twig.

So I wonder if we shouldn't try to have
```
include('template.html' , vars  , false)
```
rather than
```
include('template.html' , vars  , with_context = false)
```
and same for others options.

Reply:
To be honest I have absolutely no idea wich of those two args (context / missing) is the first one / second one .. 

Tried to follow "blindly" the doc : https://twig.symfony.com/doc/3.x/functions/include.html 

I remove them and it can be the goal of another rule i guess later

Reply:
Oh well..   we  generate "`include(...,  false, true)`" if there was only "`ignore_missing`" ? :| 

```php
function twig_include(Environment $env, $context, $template, $variables = [], $withContext = true, $ignoreMissing = false, $sandboxed = false)
{
```

Reply:
Yes, I had in mind to generate

Include ($template, [], false, true)

If the only option passed was "ignore missing".

I agree that "false" is unnecessary passed but I feel like people are not used to named parameter yet.

Reply:
Done !